graph/src/data/subgraph/schema.rs

```diff
@@ -1106,6 +1110,71 @@ impl TryFromValue for EthereumBlockHandlerFilterEntity {
     }
 }
 
+#[derive(Debug, PartialEq)]
+pub enum EthereumBlockHandlerData {
+    Block,
+    FullBlock,
+    FullBlockWithReceipts,
+}
+
+impl Default for EthereumBlockHandlerData {
+    fn default() -> EthereumBlockHandlerData {
+        EthereumBlockHandlerData::Block
+    }
+}
+
+impl FromStr for EthereumBlockHandlerData {
+    type Err = Error;
+
+    fn from_str(s: &str) -> Result<EthereumBlockHandlerData, Error> {
+        match s {
+            "FullBlock" => Ok(EthereumBlockHandlerData::FullBlock),
+            "Block" => Ok(EthereumBlockHandlerData::Block),
+            _ => Err(format_err!(
+                "failed to parse `{}` as EthereumBlockHandlerData",
+                s
+            )),
+        }
+    }
+}
+
+impl From<EthereumBlockHandlerData> for String {
+    fn from(data: EthereumBlockHandlerData) -> String {
+        match data {
+            EthereumBlockHandlerData::FullBlockWithReceipts => "FullBlockWithReceipts".into(),
+            EthereumBlockHandlerData::FullBlock => "FullBlock".into(),
+            EthereumBlockHandlerData::Block => "Block".into(),
+        }
+    }
+}
+
+impl From<super::BlockHandlerData> for EthereumBlockHandlerData {
```

Comment:
This is also missing `FullBlockWithReceipts`. I think the usage of `_` here may be counterproductive if new variants are added.